src/_P137_AXP192.ino

```diff
@@ -1,14 +1,15 @@
 #include "_Plugin_Helper.h"
 #ifdef USES_P137
 
+# ifdef ESP32
+
 // #######################################################################################################
 // #################################### Plugin 137: AXP192 Powermanagement ###############################
 // #######################################################################################################
 
 /**
  * Changelog:
  * 2022-08-24 tonhuisman: Remove [TESTING] tag, move include for _Plugin_helper.h to correct line
- *                        Removed ESP32 requirement from builds
```

Comment:
Looks like it still is ESP32 only??

Reply:
https://github.com/letscontrolit/ESPEasy/pull/4124/files#diff-55f602a56fa4564e2317e63c1108e514f83670ffb26d7500d6c058c1cd3ea5a9R19
That was the original starting point, despite some quick effort it's not feasible to try to get it built for ESP8266, other than pulling the library local and modifying it.
Also I have not yet found any ESP8266 boards that use this chip, so it's not really worth the trouble.